cluster/state/helpers.go

```diff
@@ -0,0 +1,68 @@
+// Copyright © 2022-2023 Obol Labs Inc. Licensed under the terms of a Business Source License 1.1
+
+package state
+
+import (
+	k1 "github.com/decred/dcrd/dcrec/secp256k1/v4"
+	ssz "github.com/ferranbt/fastssz"
+
+	"github.com/obolnetwork/charon/app/errors"
+	"github.com/obolnetwork/charon/app/k1util"
+)
+
+// hashRoot hashes a ssz root hasher object.
+func hashRoot(hasher rootHasher) ([32]byte, error) {
+	hw := ssz.DefaultHasherPool.Get()
+	defer ssz.DefaultHasherPool.Put(hw)
+
+	if err := hasher.HashTreeRootWith(hw); err != nil {
+		return [32]byte{}, err
+	}
+
+	resp, err := hw.HashRoot()
+	if err != nil {
+		return [32]byte{}, errors.Wrap(err, "hash root")
+	}
+
+	return resp, nil
+}
+
+// verifyEmptySig verifies that the signed mutation isn't signed.
```

Comment:
nit
```suggestion
// verifyEmptySig returns an error if the signed mutation isn't signed.
```

Reply:
your change flips the logic which isn't correct.